### PR TITLE
Content: Updates for science-of-connection page

### PR DESCRIPTION
**Pull request by shredtechular**
This PR is a partial fix for #12 .  

The only item in the issue that is not completed is:

> Below "A Spark was Lit" section, please add an "Additional Resources section" and include the following links
American Carnage article
PDF's in the external lit research folder of the shared google drive
[ ] - I have requested additional information to complete that item.

All of the other items were completed, however, and the screen cap below illustrates how these changes look in my local dev environment:

<kbd>
screen shot 2017-10-09 at 9 11 27 pm
</kbd>

### CODE REVIEW

**Comment by howtohumanbetter** [comment]
Hey Andrew! Jennifer Nicolaisen here; thank you so much for getting started with this. I'm really excited to have you on the team! Thanks for your patience with my delayed response this week; I've been out of town the past two days. It will be easier for me to communicate with you in Slack, and then I can get these documents to you ASAP.

**Comment by shredtechular** [comment]
Thanks @howtohumanbetter !  I'm in Slack now, so happy to chat/review things in there now too!

**Comment by shredtechular** [comment]
@j00bar this PR is ready to be reviewed now.

File: science-of-connection.html [hunk 1]
```diff
@@ -34,22 +34,57 @@
...
+            <div class="col-md-8 col-md-offset-2 bottommargin text-center">
+              <div class="team team-list clearfix">
+                <div class="team-image">
+                  <img src="images/rachel-headshot.jpg" alt="Rachel Wurzman, PhD, Chief Science Officer">
+                </div>
+                <div class="team-desc">
+                  <div class="team-title"><h4>Rachel Wurzman, PhD</h4><span>Chief Science Officer </span></div>
+                  <div class="team-content">
+                    <p>Background: Neuroscience PhD, post-doctoral research fellow in the LCNS at the University of Pennsylvania</p>
+                    <p>R&R: development and publication of academic research exploring the neuroscience of addiction behaviors& patterns.</p>
```
Comment (j00bar) [inline science-of-connection.html]: Do you mind turning this into an HTML entity? And check the spacing toward the end. Thanks!

Comment (shredtechular) [reply science-of-connection.html]: No worries at all, I converted all the Ampersands now in 87ad113.

@j00bar ready for round 2 of review.

**Pull request merged**